lib/trace-writer.js

```diff
@@ -40,7 +42,10 @@ function TraceWriter(logger, config) {
   this.config_ = config;
 
   /** @private {function} authenticated request function */
-  this.request_ = utils.authorizedRequestFactory(SCOPES);
+  this.request_ = utils.authorizedRequestFactory(SCOPES, {
+    credentials: config.credentials,
+    keyFile: config.keyFile || config.keyFilename
```

Comment:
Will passing in a `null` value for `keyFile` result in `GOOGLE_APPLICATION_CREDENTIALS` being used?